mpifileutils.spec

```
@@ -243,6 +248,12 @@ done
 %endif
 
 %changelog
+* Fri Jul 16 2021 Danielle M. Sikich <danielle.sikich@intel.com> - 0.11-8
+- Update to patch 67f44ad
+
+* Tue Jul 13 2021 Danielle M. Sikich <danielle.sikich@intel.com> - 0.11-7
+- Update to patch 8a22a2d
+
```

From the outside looking in shouldn't this just be the following?
```suggestion
* Fri Jul 16 2021 Danielle M. Sikich <danielle.sikich@intel.com> - 0.11-8
- Update to patch 67f44ad

```

---

> From the outside looking in shouldn't this just be the following?

@phender makes sense, updated.